.node-version

```diff
@@ -1 +1 @@
-18.16.1
+20.4.0
```

Comment:
ts-node-test is failed, looks related to https://github.com/meyfa/ts-node-test/issues/64#issuecomment-1540021495

Reply:
https://github.com/nodejs/node/issues/47880 👀

Reply:
https://github.com/kachick/renovate-config-asdf/pull/1028 may resolve this problem